### ISSUE TITLE
Failed to require the main module of 'Remote-FTP' because it requires one or more incompatible native modules (binding, binding, binding, binding, binding)

### ISSUE THREAD

**Issue by dkoppstein** [icetee/remote-ftp#802]
```
<embedded>:68906 Failed to require the main module of 'Remote-FTP' because it requires one or more incompatible native modules (binding, binding, binding, binding, binding).
Run `apm rebuild` in the package directory and restart Atom to resolve.
```

Ran `apm rebuild` in `~/.atom/packages` to no effect. Also tried uninstalling all other packages to no effect. 

```
➜  atom --version
Atom    : 1.17.0
Electron: 1.3.15
Chrome  : 52.0.2743.82
Node    : 6.5.0
```

OS X Sierra 10.12.4.

**Comment by MrVanosh** [comment]
same issue at macOS Sierra 10.12.4

**Comment by creator4ever** [comment]
In my case, error occurred on building fsevents module.
It caused by file fsevents/build/config.gypi set only default_configuration.
So, I open packages/Remote-FTP/node-modules/fsevents/package.json
and fix the line 55
```
"module_path": "./lib/binding/{configuration}/{node_abi}-{platform}-{arch}/",
```
to 
```
"module_path": "./lib/binding/{default_configuration}/{node_abi}-{platform}-{arch}/",
```

Then, click rebuild package button in debug mode.

**Comment by chobitsky** [comment]
The same symptoms occurred, I tried rebuild but it did not solve it.
I am in trouble because I do not know what is the cause.

MacOS Sierra
Atom uses stable version.

**Comment by icetee** [comment]
Try `apm rebuild` or in Atom incompatible-packages view click rebuild button.

**Comment by icetee** [comment]
Solved here [#839](https://github.com/mgrenier/remote-ftp/issues/839#issuecomment-309371225)
